README.md

```diff
@@ -13,26 +13,28 @@ This list contains only stage 1 proposals and higher that have not yet been with
+|  | [Asynchronous Iteration][async-iteration]                        | Domenic Denicola                                | [:construction:](https://github.com/tc39/test262/issues/923)      |
+|  | [`import()`][dynamic-import]                                     | Domenic Denicola                                | [:question:](https://github.com/tc39/test262/issues/1164)         |
+|  | [RegExp Lookbehind Assertions][lookbehind]                       | Daniel Ehrenberg                                | [:white_check_mark:](https://github.com/tc39/test262/issues/999)  |
+|  | [RegExp Unicode Property Escapes][unicode-escapes]               | Brian Terlson, Daniel Ehrenberg, Mathias Bynens | [:white_check_mark:](https://github.com/tc39/test262/issues/1014) |
+|  | [RegExp named capture groups][named-groups]                      | Daniel Ehrenberg, Brian Terlson                 | [:white_check_mark:](https://github.com/tc39/test262/issues/998)  |
+|  | [`s` (`dotAll`) flag for regular expressions][dot-all]           | Mathias Bynens, Brian Terlson                   | [:white_check_mark:](https://github.com/tc39/test262/issues/997)  |
+|  | [Legacy RegExp features in JavaScript][regexp-legacy]            | Mark Miller, Claude Pache                       | [:white_check_mark:](https://github.com/tc39/test262/issues/1165) |
+|  | [`Promise.prototype.finally`][finally]                           | Jordan Harband                                  | [:construction:](https://github.com/tc39/test262/issues/866)      |
+|  | [BigInt][bigint]                                                 | Daniel Ehrenberg                                | [:construction:](https://github.com/tc39/test262/issues/1056)     |
+|  | [Optional catch binding][optional-catch]                         | Michael Ficarra                                 | [:white_check_mark:](https://github.com/tc39/test262/issues/1166) |
+|  | [`import.meta`][import-meta]                                     | Domenic Denicola                                | [:question:](https://github.com/tc39/test262/issues/1342)         |
+|  | [Private methods and accessors][private-methods]                 | Daniel Ehrenberg, Kevin Gibbons                 | [:question:](https://github.com/tc39/test262/issues/1343)         |
+|  | [Array.prototype.flatMap/flatten][flat]                          | Brian Terlson, Michael Ficarra                  | [:question:](https://github.com/tc39/test262/issues/new)          |
+|  | [Numeric separators][numeric_separators]                         | Sam Goto, Rick Waldron                          | [:construction:](https://github.com/tc39/test262/issues/1051)     |
+|  | [Class Public Instance Fields & Private Instance Fields & Methods][class-fields] | Daniel Ehrenberg, Jeff Morrison | [:question:](https://github.com/tc39/test262/issues/1161)         |
```

Comment:
not a huge deal, but it'd be good to have these test262 links footnoted at the bottom, instead of inline.

Reply:
Hey, sorry I didn't see this comment until now! 🙈 Happy to make this change if you'd like – should I open up an new PR?

Reply:
Sure, thanks!

Reply:
Done: https://github.com/tc39/proposals/pull/96